src/components/map/Map.jsx

```diff
@@ -0,0 +1,41 @@
+import { MapContainer, Marker, TileLayer } from "react-leaflet";
+import L from "leaflet";
+import "leaflet/dist/leaflet.css";
+import "./map-wrapper.css";
+
+function Map({ mapData, zoomControl, additionalClass = "" }) {
+  return (
+    <MapContainer
+      center={[56.15355732197891, 10.213148468411132]} // Aarhus <3
+      zoom={14}
+      className={additionalClass}
+      zoomControl={zoomControl}
+      scrollWheelZoom={false}
+    >
+      <TileLayer
+        attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+      />
+      {mapData.map(({ latitude, longitude }, index) => (
+          <Marker
+          key={index}
+          position={[latitude, longitude]}
+          /* Using the index for numbering only works if the data they are in the correct order */
```

Comment:
Use comments consistently:

```suggestion
          // Using the index for numbering only works if the data they are in the correct order
```